src/Integration.UnitTests/Binding/BindingControllerTests.cs

```diff
@@ -66,28 +68,32 @@ public class BindingControllerTests
         public void TestInitialize()
         {
             KnownUIContextsAccessor.Reset();
-            this.sonarQubeService = new Mock<ISonarQubeService>();
```

Comment:
There are lot of stylistic changes which make it difficult to focus on the real changes. Reviewing would be easier if they were done in separate commit. Also, something as straightforward as removing `this` everywhere doesn't really need to be reviewed, so it doesn't have to form part of the review.

Reply:
You're right, my apologies.